src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs

```diff
@@ -30,6 +30,14 @@ public class HandshakeRequestMessage : ServiceMessage
         /// </summary>
         public int Version { get; set; }
 
+        public const int TypeDefault = 0;
+        public const int TypeOnDemand = 0;
+        public const int TypeWeak = 0;
```

Comment:
Change to enum type like https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServerConnectionType.cs ?